static/src/js/components/Tour/TourSteps.jsx

```diff
@@ -178,6 +187,27 @@ const TourSteps = ({
             Available filters include keyword search, spatial and temporal bounds,
             and advanced search options.
           </p>
+          <p className="search-tour__hotkey-info align-middle">
+            <strong>Tip:</strong>
+            {' '}
+            Use
+            <kbd className="mx-1 align-bottom">
+              <EDSCIcon
+                className="align-middle"
```

Comment:
Played with this a little more and I think we can get rid of the `align-middle` class on the icons.